stage1/usr_from_kvm/files.mk

```diff
@@ -0,0 +1,33 @@
+$(call setup-stamp-file,UFKF_STAMP)
+UFKF_DIR := $(MK_SRCDIR)/files
+UFKF_F1_IN_ACI := $(ACIROOTFSDIR)/etc/passwd
+UFKF_SRC_F1 := $(UFKF_DIR)/passwd
+UFKF_F2_IN_ACI := $(ACIROOTFSDIR)/etc/shadow
+UFKF_SRC_F2 := $(UFKF_DIR)/shadow
+UFKF_F3_IN_ACI := $(ACIROOTFSDIR)/usr/lib64/systemd/system/sshd.service
+UFKF_SRC_F3 := $(UFKF_DIR)/sshd.service
+UFKF_F4_IN_ACI := $(ACIROOTFSDIR)/etc/group
+UFKF_SRC_F4 := $(UFKF_DIR)/group
+UFKF_F5_IN_ACI := $(ACIROOTFSDIR)/etc/ssh/sshd_config
+UFKF_SRC_F5 := $(UFKF_DIR)/sshd_config
```

Comment:
I see that all required files are directly in `$(UFKF_DIR)`, so to avoid repetition I would do something like this:

```
$(call setup-stamp-file,UFKF_STAMP)
UFKF_DIR := $(MK_SRCDIR)/files
UFKF_VAR_RUN := $(ACIROOTFSDIR)/var/run
UFKF_ACI_FILES := \
	$(ACIROOTFSDIR)/etc/passwd \
	$(ACIROOTFSDIR)/etc/shadow \
	$(ACIROOTFSDIR)/usr/lib64/systemd/system/sshd.service \
	$(ACIROOTFSDIR)/etc/group \
	$(ACIROOTFSDIR)/etc/ssh/sshd_config
UFKF_SRC_FILES := $(addprefix $(UFKF_DIR)/,$(notdir $(UFKF_ACI_FILES)))

STAGE1_STAMPS += $(UFKF_STAMP)
STAGE1_INSTALL_FILES += $(join $(addsuffix :,$(UFKF_SRC_FILES)),$(addsuffix :-,$(UFKF_ACI_FILES)))
STAGE1_INSTALL_DIRS += \
	$(addsuffix :0755,$(sort $(call to-dir,$(UFKF_ACI_FILES)))) \
	$(UFKF_VAR_RUN):0755

$(UFKF_STAMP): $(UFKF_ACI_FILES) | $(UFKF_VAR_RUN)
	touch "$@"

$(call undefine-namespaces,UFKF)
```

I hope I didn't make any typos there. ;)